src/dimension_slice.c

```diff
@@ -920,7 +920,9 @@ ts_dimension_slice_oldest_valid_chunk_for_reorder(int32 job_id, int32 dimension_
 
 typedef struct CompressChunkSearch
 {
-	int32 chunk_id;
+	List *chunk_ids; /* list of chunk ids that match search */
+	int32 maxchunks; /*max number of chunks to return */
+	int32 chcnt;
```

Comment:
Isn't `chcnt` identical to `list_length(chunk_ids)`?

Reply:
yes, removed redundant var.